pkg/build/api/validation/validation.go

```diff
@@ -82,6 +82,15 @@ func ValidateBuildConfig(config *buildapi.BuildConfig) field.ErrorList {
 		fromRefs[fromKey] = struct{}{}
 	}
 
+	schedulePolicy := config.Spec.SchedulePolicy
+	if schedulePolicy != nil {
+		switch *schedulePolicy {
+		case buildapi.BuildSchedulePolicyParallel, buildapi.BuildSchedulePolicySerial, buildapi.BuildSchedulePolicySerialLatestOnly:
+		default:
+			allErrs = append(allErrs, field.Invalid(specPath.Child("schedulePolicy"), *schedulePolicy, "schedule policy must parallel, latest-only, or serial"))
+		}
```

Comment:
this will need to be updated to force schedulePolicy to be non-nil once we're defaulting it in the api.